extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -187,10 +187,14 @@ export class AmpAdUIHandler {
 
   /**
    * @param {string} name
-   * @return {!Element}
+   * @return {?Element}
```

Comment:
We call `addDefaultUiComponent_` in `#createPlaceholderCallback` which returns null by default. I don't think we need extra assertion check.